src/main/webapp/js/controllers.js

```diff
@@ -48,28 +48,34 @@ angular
                     };
 
                     function mockList() {
-                        var descr = "Mussum ipsum cacilds, vidis litro abertis. Consetis adipiscings elitis. Pra lá , depois divoltis porris, paradis. Paisis, filhis, espiritis santis.";
+                        var descr = "Mussum ipsum cacilds, vidis litro abertis. Consetis adipiscings elitis. Pra lÃ¡ , depois divoltis porris, paradis. Paisis, filhis, espiritis santis.";
```

Comment:
My enconding configurations are ok, with "Text File: UTF-8" and "New Line: Unix". Anyway  i fixed the line and commited again.